docs/cli/standalone-cli.md

```
@@ -0,0 +1,54 @@
+# Standalone ClickHouse Monitoring CLI (Rust)
+
+`tools/ch-monitor-cli` provides a standalone CLI that talks to the existing API.
+
+## Why these libs
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)